docs/HowTo/Find-and-Connect/Specifying-NAT.md

```diff
@@ -19,6 +19,25 @@ While Hyperledger Besu is running, the following are not supported:
 * Changing NAT methods. To change the NAT method, restart the node with the
   [`--nat-method`](../../Reference/CLI/CLI-Syntax.md#nat-method) option set.
 
+## Auto
+
+`AUTO` is the default NAT method. `AUTO` detects if Besu is running inside a Kubernetes cluster or
+a Docker container. If running inside:
+
+* A Kubernetes cluster, `AUTO` acts as if you specified [`KUBERNETES`](#kubernetes).
```

Comment:
maybe rename the three to `AUTO` is set to `KUBERNETES`? ...

Reply:
I think we probably need add a little one or two lines re what that is doing behind the scenes? In K8S the NAT mode then uses the POD IP as .... If this is doco'd somewhere else a link should do. Similar for the Docker options etc